project/src/main/scala/edu/gemini/osgi/tools/app/RPMDistHandler.scala

```diff
@@ -82,17 +79,70 @@ class RPMDistHandler(jre: Option[String]) extends DistHandler {
     val ret = proc.waitFor
     if (ret != 0) sys.error("tar returned " + ret)
 
-    // Now create the RPM
+    // The RPM spec file.
     val specPath = new File(wd, "rpm.spec")
-    val rpmArgs = Array("rpmbuild", "-bb", s"${specPath.getAbsolutePath}")
-    val result = Runtime.getRuntime.exec(rpmArgs).waitFor()
-    if (result != 0) {
-      println("*** " + rpmArgs.mkString(" "))
-      println("*** rpmbuild returned " + result)
+
+    // Set up an SSH connection to the Linux VM where rpmbuild will be run.
+    if (config.remoteBuildInfo.isEmpty)
+      log.error("no remote build info: update OcsCredentials?")
```

Comment:
No, it won't, but I was counting on the exception being thrown in the next line via the `None.get` call to do that. Do you think this should be handled differently? It seems that allowing exceptions to bubble up is often the way that things are terminated in the build process.